code/modules/client/preferences_mysql.dm

```diff
@@ -57,7 +56,7 @@
 					UI_style='[UI_style]',
 					UI_style_color='[UI_style_color]',
 					UI_style_alpha='[UI_style_alpha]',
-					be_special='[be_special]',
+					be_role='[sql_sanitize_text(list2params(be_special))]',
```

Comment:
AAaahh yes, sql_sanitize strips semicolons, because semicolons are bad in SQL queries
However, as you can't have a custom be_role outside of pressing buttons, you can just remove this entirely